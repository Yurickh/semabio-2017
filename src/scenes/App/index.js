import React, { Component } from 'react'
import Home from '../Home'
import About from '../About'
import Schedule from '../Schedule'
import Features from '../Features'
import Shop from '../Shop'
import Faq from '../Faq'
import Map from '../Map'
import Contact from '../Contact'
import PlacesToStay from '../PlacesToStay'
import Partners from '../Partners'

import BuyShirt from '../BuyShirt'
import BuyPackage from '../BuyPackage'

import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'

const SPA = (props) => (
  <div className="App">
    <Home {...props} />
    <About {...props} />
    <Schedule {...props} />
    <Features {...props} />
    <Shop {...props} />
    <Faq {...props} />
    <Map {...props} />
    <PlacesToStay {...props} />
    <Contact {...props} />
    <Partners {...props} />
  </div>
)

class App extends Component {
  render() {
    return (
      <Router>
        <Switch>
          {/* <Route path="/comprar/camiseta" component={BuyShirt} />
          <Route path="/comprar/:product/:checkoutId" component={BuyShirt} />
          <Route path="/comprar/:product" component={BuyPackage} /> */}
          <Route exact path="/" component={SPA} />
        </Switch>
      </Router>
    )
  }
}

export default App
