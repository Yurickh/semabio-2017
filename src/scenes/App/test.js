import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import App from '../App'

describe('App', () => {
  it('should render correctly', () => {
    const page = shallow(<App />)
    expect(toJSON(page)).toMatchSnapshot()
  })
})
