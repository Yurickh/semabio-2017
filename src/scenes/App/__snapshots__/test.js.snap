// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
  </Switch>
</BrowserRouter>
`;
